docs/4.0/getting-started/download.md

```diff
@@ -68,13 +68,13 @@ Bootstrap's `package.json` contains some additional metadata under the following
 Install Bootstrap in your Ruby apps using [Bundler](https://bundler.io/) (**recommended**) and [RubyGems](https://rubygems.org/) by adding the following line to your [`Gemfile`](https://bundler.io/gemfile.html):
 
 {% highlight ruby %}
-gem 'bootstrap', '~> 4.0.0.alpha6'
+gem 'bootstrap', '~> 4.0.0.beta'
```

Comment:
here it is: `gem 'bootstrap', '~> 4.0.0.beta'`
https://github.com/twbs/bootstrap-rubygem/blob/master/README.md